### ISSUE TITLE
False positive on Quartz 2.3.0

### ISSUE THREAD

**Issue by bopinto** [dependency-check/DependencyCheck#1579]
I believe the plugin is reporting a false positive on the quartz library, version 2.3.0 - reported as cpe:/a:jenkins:jenkins:2.3

```xml
<dependency>
    <groupId>org.quartz-scheduler</groupId>
    <artifactId>quartz</artifactId>
    <version>2.3.0</version>
</dependency>
```

**Comment by sellersj** [comment]
Work around suppression.

```xml
<suppress>
  <gav regex="true">^org\.quartz-scheduler:quartz:.*$</gav>
  <cpe>cpe:/a:jenkins:jenkins</cpe>
</suppress>

**Comment by TobiX** [comment]
This is a fallout of the lucene update discussed in #1580

**Comment by jeremylong** [comment]
This issue was resolved with 4.0.1.
